pkgs/development/python-modules/path/default.nix

```
@@ -12,12 +12,12 @@
 }:
 
 buildPythonPackage rec {
-  pname = "path.py";
-  version = "12.0.1";
+  pname = "path";
+  version = "15.1.2";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "9f2169633403aa0423f6ec000e8701dd1819526c62465f5043952f92527fea0f";
+    sha256 = "u2Ka79hoJb8hyLz6D4aRpuWr3D5D1Qtib+aqxbE/YLc=";
   };
 
   checkInputs = [ pytestCheckHook pytest-flake8 glibcLocales packaging ];
```

```suggestion
  checkInputs = [ pytestCheckHook glibcLocales packaging ];
```
we shouldn't do lint tests.